modules/ztp-deploying-a-site.adoc

```diff
@@ -182,6 +182,6 @@ The `SiteConfig` CR creates the following CRs on the hub cluster:
 * `ClusterDeployment`
 * `InfraEnv`
 * `NMStateConfig` - One per node
-* `ExtraManifestsConfigMap` - Extra manifests. The additional manifests include workload partitioning, mountpoint hiding, sctp enablement, and more. The extra manifests are automatically merged into a single manifest per each `MachineConfigPool` role, which is named as `predefined-extra-manifests-<role>`.
+* `ExtraManifestsConfigMap` - Extra manifests. The extra manifests include workload partitioning, mountpoint hiding, sctp enablement, and more. To automatically merge the extra manifests into a single manifest per each `MachineConfigPool` role, which is named as `predefined-extra-manifests-<role>`, set the `mergeDefaultMachineConfigs` to `true` in the `SiteConfig.yaml` file.
```

Comment:
For a little more clarification...the flag  `mergeDefaultMachineConfigs` would go under `.spec.clusters.mergeDefaultMachineConfigs` in SiteConfig. Otherwise it looks good!